docs/src/fock.md

```diff
@@ -0,0 +1,147 @@
+# Fock States
+
+```@meta
+DocTestSetup = quote
+    using QuantumSymbolics, QuantumOptics
+end
+```
+
+In this section, we describe symbolic representations of Fock states in QuantumSymbolics, which can be numerically translated to `QuantumOptics.jl`. A Fock state is the number state representation $|n\rangle$ of a system with $n$ particles. 
+
+One can define a basis of the Fock space with `FockBasis(N, offset=0)`, where `N` is the highest available Fock state and `offset` is the lowest cutoff state, which is by default zero. In the following example, we create a `FockState` with 3 quanta in an infinite-dimension Fock space:
```

Comment:
The symbolic objects probably should always use `FockBasis(Inf)`. The non-inf is for cutoffs when doing numerical representations, where it has to be finite. That should be mentioned here and addressed below where we talk about expressing numerically.

Reply:
OK that sounds good to me.